Makefile.js

```diff
@@ -71,8 +71,7 @@ const NODE = "node ", // intentional extra space
     // Files
     RULE_FILES = glob.sync("lib/rules/*.js").filter(filePath => path.basename(filePath) !== "index.js"),
     JSON_FILES = find("conf/").filter(fileType("json")),
-    MARKDOWNLINT_IGNORED_FILES = fs.readFileSync(path.join(__dirname, ".markdownlintignore"), "utf-8").split("\n"),
-    MARKDOWN_FILES_ARRAY = find("docs/").concat(ls(".")).filter(fileType("md")).filter(file => !MARKDOWNLINT_IGNORED_FILES.includes(file)),
+    MARKDOWNLINT_IGNORED = fs.readFileSync(path.join(__dirname, ".markdownlintignore"), "utf-8").split("\n"),
```

Comment:
Content of `.markdownlintignore` are gitignore patterns. We can use the `ignore` library we already have as a dependency to filter out ignored files. That would be easier and more reliable than a custom implementation.

Ideally, we would use a glob library that supports gitignore patterns so that it doesn't traverse ignored directories at all, but that could be another enhancement.

And I'm not sure why are we using markdownlint API here instead of markdownlint CLI which could handle `.markdownlintignore` by itself.

Reply:
can I try adding the `glob` library or can we use markdownlint CLI instead of markdownlint API?

Reply:
I would try with markdownlint CLI. That looks a lot simpler and I don't see any specific reasons to use API for our use case.

Reply:
Sure, I will update.